localisation/00_zuwc_united_localisation.csv

```diff
@@ -3024,6 +3020,10 @@ RULE_ADOPTION_NO_PLAYER_DYNASTY;Restricted;;;;;;;;;;;;;x
 RULE_ADOPTION_NO_PLAYER_DYNASTY_DESC;Only players can adopt newly-spawned characters by adoption event into the player's dynasty.;;;;;;;;;;;;;x
 RULE_ADOPTION_DISABLED;Disabled;;;;;;;;;;;;;x
 RULE_ADOPTION_DISABLED_DESC;Adoption event is disabled.;;;;;;;;;;;;;x
+mind_control_1;Strong Mind Control;Contr�le mental fort;Starke Gedankenkontrolle;;;;;;;;;;;x
+mind_control_2;Weaking Mind Control;;;;;;;;;;;;;x
```

Comment:
Maybe weakening?
Also, this choice is weird, it implies that mind control power is being weakening, but what if it actually growing stronger?